src/main/java/uk/gov/hmcts/reform/idam/client/CoreFeignConfiguration.java

```diff
@@ -8,12 +8,9 @@
 import org.springframework.boot.autoconfigure.http.HttpMessageConverters;
 import org.springframework.cloud.openfeign.support.SpringEncoder;
 import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Primary;
 import org.springframework.context.annotation.Scope;
 
-
-@Configuration
```

Comment:
Is this expected?

Reply:
Yes - we want to inherit the Configuration of the host project, not have our own here, so we can apply the config from there.

Reply:
Also, this is Feign config, so would not apply to Spring...